components/ws-manager-mk2/controllers/status.go

```diff
@@ -188,10 +188,12 @@ func updateWorkspaceStatus(ctx context.Context, workspace *workspacev1.Workspace
 }
 
 func isDisposalFinished(ws *workspacev1.Workspace) bool {
+	c := wsk8s.GetCondition(ws.Status.Conditions, string(workspacev1.WorkspaceConditionContentReady))
+	noContentInit := c == nil || c.Status == metav1.ConditionFalse
```

Comment:
fixes workspaces stuck in Stopping when the ContentReady condition never got added (e.g. a failed workspace during startup)